packages/build/src/main.ts

```diff
@@ -2,14 +2,12 @@ import * as fs from "fs"
 import * as path from "path"
 import * as webpack from "webpack"
 
-import log from "../log"
+import log from "../../core/src/log"
```

Comment:
Since core is a separate package that might not be installed alongside this package we'll need to either extract the log function to a common dependency, copy it here, or drop it. Since this module already mixes `console.log` and `log` I think we can do the simplest thing and replace the two `log` calls with `console.log` and break the cycle.